webui/react/src/components/kit/Columns.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { ReactNode } from 'react';
+
+import css from './Columns.module.scss';
+
+interface ColumnProps {
+  children: ReactNode;
+  align?: 'left' | 'center' | 'right';
+}
+
+interface ColumnsProps {
+  children: ReactNode;
+  gap?: number;
```

Comment:
question: do we really want to allow arbitrary gaps? it might make sense to make this something like `0 | 8 | 16`, but I'll defer to Claire or design

Reply:
Good call.